crates/shared/src/sources.rs

```diff
@@ -28,6 +28,18 @@ arg_enum! {
     }
 }
 
+pub fn defaults_for_chain(chain_id: u64) -> Vec<BaselineSource> {
+    match chain_id {
+        1 | 4 => vec![
+            BaselineSource::UniswapV2,
+            BaselineSource::SushiSwap,
+            BaselineSource::BalancerV2,
+        ],
+        100 => vec![BaselineSource::Honeyswap, BaselineSource::SushiSwap],
+        _ => panic!("unsupported chain {:#x}", chain_id),
```

Comment:
Nit: Wouldn't empty Vec be more appropriate? Then we can react however we want to in main.

Reply:
Yeah, I guess these hidden panic's aren't great. I will change it to a `Result<Vec<BaselineSource>>`.